spec/controllers/settings_controller_spec.rb

```diff
@@ -1,5 +1,4 @@
 require 'spec_helper'
 
 describe SettingsController, type: :controller do
-
```

Comment:
Feel free to write a settings controller test as well! We'll go over it in class tomorrow ;)